src/components/selection_input/selection_input.ts

```diff
@@ -179,12 +189,19 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
   }
 
   getColor(range: SelectionRange) {
+    if (!this.isEnabled(range)) {
+      return "#fff";
```

Comment:
why would we want the input to be white ? And `style="#fff"` isn't valid CSS so this does nothing